app/lib/theme/widgets/NumberWidget.tsx

```diff
@@ -29,7 +42,7 @@ export const MoneyWidget: React.FC<WidgetProps> = ({
           ) {
```

Comment:
PercentageWidget additionally had a check to make sure the number was between 0-100, but I think there could be cases (e.g. emissions calculations) where the value might be outside of 0-100, so I took it out.

Reply:
Very nice! Percentage values aren't necessarily bound by 0-100, and that concern should be up to the user of the widget instead.